variables.md

```diff
@@ -49,6 +49,7 @@ EXTRABOOTPARAMS_DELETE_CHARACTERS | string | | Characters to delete from boot pr
 EXTRABOOTPARAMS_DELETE_NEEDLE_TARGET | string | | If specified, go back with the cursor until this needle is matched to delete characters from there. Needs EXTRABOOTPARAMS_BOOT_LOCAL and should be combined with EXTRABOOTPARAMS_DELETE_CHARACTERS.
 EXTRATEST | boolean | false | Enables execution of extra tests, see `load_extra_tests`
 FLAVOR | string | | Defines flavor of the product under test, e.g. `staging-.-DVD`, `Krypton`, `Argon`, `Gnome-Live`, `DVD`, `Rescue-CD`, etc.
+FORMULAS_PATH | string | | It is used to set the uri of the tar file in the configuration_management for salt-formulas
```

Comment:
Can you tell me what should i check in variables.md AUTOYAST? i assume that the description is not descriptive enough. right? better tell me what do you think it should be and take it from there. because i am also not sure what the description of AUTOYAST can provide more accurate here

Reply:
My suggestion would be something like:
```
SALT_FORMULAS        | string    |               | Relative path to [data directory of os-autoinst-distri-opensuse repo](https://github.com/os-autoinst/os-autoinst-distri-opensuse/tree/master/data) containing a tarball with all the needed files (top.sls, form.yml, ...) to support provisioning with Salt masterless mode.
```
it admits puppet too, but let's not overcomplicated if we only use salt.
wdyt?

Reply:
changed.